package/ffi/sdks/dart.go

```diff
@@ -109,6 +109,19 @@ func (s *DartSDK) Build(ctx context.Context, client *dagger.Client, container *d
 		}
 	}
 
+	// run strip_xcframework.sh
+	// this is because the XCFramework is not stripped and is too large to upload to pub.dev
+	// TODO: we should do this when we build the XCFramework in Actions but for now this is a quick fix
```

Comment:
now that I added this to the action, we should remove this code before creating the next engine release